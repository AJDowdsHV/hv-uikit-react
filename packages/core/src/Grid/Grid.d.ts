import { GridProps, StandardProps } from "@material-ui/core";

export type Spacing = "xs" | "sm" | "md" | "lg" | "xl" | "auto";
export type Columns = 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12;

export interface HvGridProps extends StandardProps<GridProps, "", "spacing"> {
  /**
   * Defines the space between the type item component. It can only be used on a type container component.
   * Based in the 7.5px factor defined in the theme| it allows the definition of this factor based on the factor
   * (number between 0 and 10)| breakpoint or auto.
   */
  spacing?: Spacing & Columns;
}

export default function HvGrid(props: HvGridProps): JSX.Element | null;
