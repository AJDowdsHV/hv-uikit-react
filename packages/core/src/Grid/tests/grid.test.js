import React from "react";
import { mount } from "enzyme";

import HvProvider from "../../Provider";
import Grid from "..";

window.matchMedia = jest.fn(() => ({
  matches: false,
  addListener: jest.fn(),
  removeListener: jest.fn()
}));

describe("Grid withStyles", () => {
  const wrapper = mount(
    <HvProvider>
      <Grid />
    </HvProvider>
  );

  it("should be defined", () => {
    expect(wrapper).toBeDefined();
  });

  it("should render correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });

  it("should render the Grid component", () => {
    const GridComponent = wrapper.find(Grid);
    expect(GridComponent.length).toBe(1);
  });
});
