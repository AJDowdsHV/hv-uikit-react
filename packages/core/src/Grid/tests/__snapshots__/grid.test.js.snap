// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Grid withStyles should render correctly 1`] = `
<HvProvider
  changeTheme={[Function]}
  router={null}
  theme={null}
  uiKitTheme="dawn"
>
  <ThemeProvider
    theme={
      Object {
        "breakpoints": Object {
          "between": [Function],
          "down": [Function],
          "keys": Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ],
          "only": [Function],
          "up": [Function],
          "values": Object {
            "lg": 1270,
            "md": 960,
            "sm": 600,
            "xl": 1920,
            "xs": 0,
          },
          "width": [Function],
        },
        "direction": "ltr",
        "hv": Object {
          "deprecated": Object {
            "palette": Object {
              "accent": Object {},
              "atmosphere": Object {},
              "base": Object {},
              "semantic": Object {},
              "support": Object {},
            },
            "typography": Object {
              "disabledText": Object {},
            },
            "viz": Object {
              "palette": Object {
                "categorical": Object {},
                "polarizes": Object {},
                "sequential": Object {},
                "undefinedState": Object {},
              },
            },
          },
          "name": "dawn",
          "palette": Object {
            "accent": Object {
              "acce0": "#FFFFFF",
              "acce1": "#414141",
              "acce2": "#146BD2",
              "acce2h": "#4389DB",
              "acce3": "#CC0000",
            },
            "atmosphere": Object {
              "atmo1": "#FFFFFF",
              "atmo2": "#F9F9F9",
              "atmo3": "#F5F5F5",
              "atmo4": "#F0F0F0",
              "atmo5": "#DEDEDE",
              "atmo6": "#BCBCBC",
              "atmo7": "#999999",
            },
            "base": Object {
              "base1": "#FFFFFF",
              "base2": "#414141",
            },
            "semantic": Object {
              "sema1": "#63A621",
              "sema10": "#EF9A9A",
              "sema11": "#E57373",
              "sema12": "#EF5350",
              "sema13": "#E53935",
              "sema14": "#C62828",
              "sema15": "#80DEEA",
              "sema16": "#4DD0E1",
              "sema17": "#26C6DA",
              "sema18": "#00ACC1",
              "sema19": "#00838F",
              "sema2": "#72CCCB",
              "sema3": "#E68C17",
              "sema4": "#FF5252",
              "sema5": "#C51162",
              "sema6": "#AA00FF",
              "sema7": "#D3E3F6",
              "sema8": "#DFECD1",
              "sema9": "#F5D8D8",
            },
            "support": Object {
              "supp1": "#0F8B8D",
              "supp2": "#734B6D",
              "supp3": "#4E7599",
              "supp4": "#C19C31",
              "supp5": "#546B6B",
            },
          },
          "spacing": Object {
            "lg": 60,
            "md": 30,
            "sm": 20,
            "xl": 90,
            "xs": 10,
          },
          "type": "light",
          "typography": Object {
            "3xlTitle": Object {
              "color": "#414141",
              "fontSize": "42px",
              "fontWeight": 600,
              "letterSpacing": "0.02em",
              "lineHeight": "52px",
            },
            "4xlTitle": Object {
              "color": "#414141",
              "fontSize": "52px",
              "fontWeight": 400,
              "letterSpacing": "0.02em",
              "lineHeight": "60px",
            },
            "5xlTitle": Object {
              "color": "#414141",
              "fontSize": "52px",
              "fontWeight": 600,
              "letterSpacing": "0.02em",
              "lineHeight": "60px",
            },
            "disabledButtonText": Object {
              "color": "#999999",
              "fontSize": "14px",
              "fontWeight": 600,
              "letterSpacing": "0.02em",
              "lineHeight": "20px",
            },
            "disabledText": Object {
              "color": "#999999",
              "fontSize": "12px",
              "fontWeight": 400,
              "letterSpacing": "0.02em",
              "lineHeight": "16px",
            },
            "fontFamily": Array [
              "'Open Sans'",
              "Arial",
              "Helvetica",
              "sans-serif",
            ],
            "highlightText": Object {
              "color": "#414141",
              "fontSize": "14px",
              "fontWeight": 600,
              "letterSpacing": "0.02em",
              "lineHeight": "20px",
            },
            "infoText": Object {
              "color": "#999999",
              "fontSize": "12px",
              "fontWeight": 400,
              "letterSpacing": "0.02em",
              "lineHeight": "16px",
            },
            "inlineLink": Object {
              "color": "#146BD2",
              "fontSize": "14px",
              "fontWeight": 600,
              "letterSpacing": "0.02em",
              "lineHeight": "20px",
            },
            "lTitle": Object {
              "color": "#414141",
              "fontSize": "32px",
              "fontWeight": 400,
              "letterSpacing": "0.02em",
              "lineHeight": "40px",
            },
            "labelText": Object {
              "color": "#414141",
              "fontSize": "12px",
              "fontWeight": 600,
              "letterSpacing": "0.02em",
              "lineHeight": "16px",
            },
            "mTitle": Object {
              "color": "#414141",
              "fontSize": "22px",
              "fontWeight": 600,
              "letterSpacing": "0.02em",
              "lineHeight": "30px",
            },
            "normalText": Object {
              "color": "#414141",
              "fontSize": "14px",
              "fontWeight": 400,
              "letterSpacing": "0.02em",
              "lineHeight": "20px",
            },
            "placeholderText": Object {
              "color": "#999999",
              "fontSize": "14px",
              "fontWeight": 400,
              "letterSpacing": "0.02em",
              "lineHeight": "20px",
            },
            "sLink": Object {
              "color": "#146BD2",
              "fontSize": "12px",
              "fontWeight": 400,
              "letterSpacing": "0.02em",
              "lineHeight": "16px",
            },
            "sText": Object {
              "color": "#414141",
              "fontSize": "12px",
              "fontWeight": 400,
              "letterSpacing": "0.02em",
              "lineHeight": "16px",
            },
            "sTitle": Object {
              "color": "#414141",
              "fontSize": "22px",
              "fontWeight": 400,
              "letterSpacing": "0.02em",
              "lineHeight": "30px",
            },
            "selectedNavText": Object {
              "color": "#CC0000",
              "fontSize": "14px",
              "fontWeight": 600,
              "letterSpacing": "0.02em",
              "lineHeight": "20px",
            },
            "selectedText": Object {
              "background": "#414141",
              "color": "#F9F9F9",
              "fontSize": "14px",
              "fontWeight": 600,
              "letterSpacing": "0.02em",
              "lineHeight": "20px",
            },
            "vizText": Object {
              "color": "#414141",
              "fontSize": "10px",
              "fontWeight": 400,
              "letterSpacing": "0.02em",
              "lineHeight": "15px",
            },
            "xlTitle": Object {
              "color": "#414141",
              "fontSize": "32px",
              "fontWeight": 600,
              "letterSpacing": "0.02em",
              "lineHeight": "40px",
            },
            "xsTitle": Object {
              "color": "#414141",
              "fontSize": "18px",
              "fontWeight": 600,
              "letterSpacing": "0.02em",
              "lineHeight": "28px",
            },
            "xxlTitle": Object {
              "color": "#414141",
              "fontSize": "42px",
              "fontWeight": 400,
              "letterSpacing": "0.02em",
              "lineHeight": "52px",
            },
            "xxsTitle": Object {
              "color": "#414141",
              "fontSize": "18px",
              "fontWeight": 400,
              "letterSpacing": "0.02em",
              "lineHeight": "28px",
            },
          },
          "viz": Object {
            "palette": Object {
              "categorical": Object {
                "cviz1": "#2DB3E0",
                "cviz10": "#419393",
                "cviz11": "#58C9DD",
                "cviz12": "#F8C189",
                "cviz13": "#528BBF",
                "cviz14": "#DE64BD",
                "cviz15": "#73BF9C",
                "cviz16": "#9672C1",
                "cviz17": "#EB95AF",
                "cviz18": "#646A98",
                "cviz19": "#D88136",
                "cviz2": "#FF990D",
                "cviz20": "#309B59",
                "cviz3": "#D31F6B",
                "cviz4": "#FFD00A",
                "cviz5": "#18C38A",
                "cviz6": "#098CB2",
                "cviz7": "#EA5B86",
                "cviz8": "#9B469B",
                "cviz9": "#F4763D",
              },
              "polarizes": Object {
                "cviz21": "#869F1E",
                "cviz22": "#A4991C",
                "cviz23": "#C4931A",
                "cviz24": "#E68C17",
                "cviz25": "#F27034",
                "cviz26": "#FF5252",
                "cviz27": "#EC3D57",
                "cviz28": "#D8265D",
                "sema1": "#63A621",
                "sema5": "#C51162",
              },
              "sequential": Object {
                "cviz1": "#2DB3E0",
                "cviz1_100": "#E3F1F6",
                "cviz1_200": "#CFEAF3",
                "cviz1_300": "#BAE3F3",
                "cviz1_400": "#A6DCEE",
                "cviz1_500": "#92D5EC",
                "cviz1_600": "#7ECEE9",
                "cviz1_700": "#69C7E7",
                "cviz1_800": "#55C0E4",
                "cviz1_900": "#40B9E2",
              },
              "undefinedState": Object {
                "atmo5": "#DEDEDE",
              },
            },
          },
        },
        "mixins": Object {
          "gutters": [Function],
          "toolbar": Object {
            "@media (min-width:0px) and (orientation: landscape)": Object {
              "minHeight": 48,
            },
            "@media (min-width:600px)": Object {
              "minHeight": 64,
            },
            "minHeight": 56,
          },
        },
        "overrides": Object {
          "MuiAppBar": Object {
            "colorDefault": Object {
              "backgroundColor": "#FFFFFF",
              "contrastText": "#414141",
            },
            "root": Object {
              "@media (min-width: 600px)": Object {
                "height": "50px",
              },
              "borderTop": "4px solid #CC0000",
            },
          },
          "MuiIconButton": Object {
            "root": Object {
              "&:focus": Object {
                "outlineColor": "Highlight",
                "outlineOffset": -2,
                "outlineStyle": "auto",
                "outlineWidth": 4,
              },
              "&:hover": Object {
                "backgroundColor": "#F0F0F0",
              },
              "borderRadius": 0,
              "padding": 0,
            },
          },
          "MuiToolbar": Object {
            "dense": Object {
              "minHeight": "46px",
            },
            "gutters": Object {
              "@media (min-width:600px)": Object {
                "paddingLeft": "20px",
                "paddingRight": "20px",
              },
              "paddingLeft": "20px",
              "paddingRight": "20px",
            },
            "root": Object {
              "@media (min-width: 600px)": Object {
                "minHeight": "46px",
              },
              "alignItems": "center",
            },
          },
        },
        "palette": Object {
          "acce0": "#FFFFFF",
          "acce1": "#414141",
          "acce2": "#146BD2",
          "acce2h": "#4389DB",
          "acce3": "#CC0000",
          "action": Object {
            "activatedOpacity": 0.12,
            "active": "rgba(0, 0, 0, 0.54)",
            "disabled": "rgba(0, 0, 0, 0.26)",
            "disabledBackground": "rgba(0, 0, 0, 0.12)",
            "disabledOpacity": 0.38,
            "focus": "rgba(0, 0, 0, 0.12)",
            "focusOpacity": 0.12,
            "hover": "rgba(0, 0, 0, 0.04)",
            "hoverOpacity": 0.04,
            "selected": "rgba(0, 0, 0, 0.08)",
            "selectedOpacity": 0.08,
          },
          "atmo1": "#FFFFFF",
          "atmo2": "#F9F9F9",
          "atmo3": "#F5F5F5",
          "atmo4": "#F0F0F0",
          "atmo5": "#DEDEDE",
          "atmo6": "#BCBCBC",
          "atmo7": "#999999",
          "augmentColor": [Function],
          "background": Object {
            "default": "#F5F5F5",
            "paper": "#fff",
          },
          "base1": "#FFFFFF",
          "base2": "#414141",
          "common": Object {
            "black": "#000",
            "white": "#fff",
          },
          "contrastThreshold": 3,
          "divider": "rgba(0, 0, 0, 0.12)",
          "error": Object {
            "contrastText": "#fff",
            "dark": "#d32f2f",
            "light": "#e57373",
            "main": "#f44336",
          },
          "getContrastText": [Function],
          "grey": Object {
            "100": "#f5f5f5",
            "200": "#eeeeee",
            "300": "#e0e0e0",
            "400": "#bdbdbd",
            "50": "#fafafa",
            "500": "#9e9e9e",
            "600": "#757575",
            "700": "#616161",
            "800": "#424242",
            "900": "#212121",
            "A100": "#d5d5d5",
            "A200": "#aaaaaa",
            "A400": "#303030",
            "A700": "#616161",
          },
          "info": Object {
            "contrastText": "#fff",
            "dark": "#1976d2",
            "light": "#64b5f6",
            "main": "#2196f3",
          },
          "primary": Object {
            "contrastText": "#fff",
            "dark": "rgb(14, 74, 147)",
            "light": "#4389DB",
            "main": "#146BD2",
          },
          "secondary": Object {
            "contrastText": "#fff",
            "dark": "rgb(45, 45, 45)",
            "light": "#999999",
            "main": "#414141",
          },
          "sema1": "#63A621",
          "sema10": "#EF9A9A",
          "sema11": "#E57373",
          "sema12": "#EF5350",
          "sema13": "#E53935",
          "sema14": "#C62828",
          "sema15": "#80DEEA",
          "sema16": "#4DD0E1",
          "sema17": "#26C6DA",
          "sema18": "#00ACC1",
          "sema19": "#00838F",
          "sema2": "#72CCCB",
          "sema3": "#E68C17",
          "sema4": "#FF5252",
          "sema5": "#C51162",
          "sema6": "#AA00FF",
          "sema7": "#D3E3F6",
          "sema8": "#DFECD1",
          "sema9": "#F5D8D8",
          "success": Object {
            "contrastText": "rgba(0, 0, 0, 0.87)",
            "dark": "#388e3c",
            "light": "#81c784",
            "main": "#4caf50",
          },
          "supp1": "#0F8B8D",
          "supp2": "#734B6D",
          "supp3": "#4E7599",
          "supp4": "#C19C31",
          "supp5": "#546B6B",
          "text": Object {
            "disabled": "#999999",
            "hint": "rgba(0, 0, 0, 0.38)",
            "primary": "#414141",
            "secondary": "rgba(0, 0, 0, 0.54)",
          },
          "tonalOffset": 0.2,
          "type": "light",
          "warning": Object {
            "contrastText": "rgba(0, 0, 0, 0.87)",
            "dark": "#f57c00",
            "light": "#ffb74d",
            "main": "#ff9800",
          },
        },
        "props": Object {
          "MuiButtonBase": Object {
            "disableRipple": true,
          },
          "MuiInput": Object {
            "disableUnderline": true,
          },
        },
        "shadows": Array [
          "none",
          "none",
          "none",
          "none",
          "none",
          "none",
          "none",
          "none",
          "none",
          "none",
          "none",
          "none",
          "none",
          "none",
          "none",
          "none",
          "none",
          "none",
          "none",
          "none",
          "none",
          "none",
          "none",
          "none",
          "none",
        ],
        "shape": Object {
          "borderRadius": 0,
        },
        "spacing": [Function],
        "transitions": Object {
          "create": [Function],
          "duration": Object {
            "complex": 375,
            "enteringScreen": 225,
            "leavingScreen": 195,
            "short": 250,
            "shorter": 200,
            "shortest": 150,
            "standard": 300,
          },
          "easing": Object {
            "easeIn": "cubic-bezier(0.4, 0, 1, 1)",
            "easeInOut": "cubic-bezier(0.4, 0, 0.2, 1)",
            "easeOut": "cubic-bezier(0.0, 0, 0.2, 1)",
            "sharp": "cubic-bezier(0.4, 0, 0.6, 1)",
          },
          "getAutoHeightDuration": [Function],
        },
        "typography": Object {
          "body1": Object {
            "color": "#414141",
            "fontFamily": Array [
              "'Open Sans'",
              "Arial",
              "Helvetica",
              "sans-serif",
            ],
            "fontSize": "14px",
            "fontWeight": 400,
            "letterSpacing": "0.02em",
            "lineHeight": "20px",
          },
          "body2": Object {
            "color": "#414141",
            "fontFamily": Array [
              "'Open Sans'",
              "Arial",
              "Helvetica",
              "sans-serif",
            ],
            "fontSize": "14px",
            "fontWeight": 400,
            "letterSpacing": "0.02em",
            "lineHeight": "20px",
          },
          "button": Object {
            "color": "#414141",
            "fontFamily": Array [
              "'Open Sans'",
              "Arial",
              "Helvetica",
              "sans-serif",
            ],
            "fontSize": "14px",
            "fontWeight": 600,
            "letterSpacing": "0.02em",
            "lineHeight": "20px",
            "textTransform": "uppercase",
          },
          "caption": Object {
            "fontFamily": Array [
              "'Open Sans'",
              "Arial",
              "Helvetica",
              "sans-serif",
            ],
            "fontSize": "0.75rem",
            "fontWeight": 400,
            "lineHeight": 1.66,
          },
          "fontFamily": Array [
            "'Open Sans'",
            "Arial",
            "Helvetica",
            "sans-serif",
          ],
          "fontSize": 14,
          "fontWeightBold": 700,
          "fontWeightLight": 300,
          "fontWeightMedium": 500,
          "fontWeightRegular": 400,
          "h1": Object {
            "color": "#414141",
            "fontFamily": Array [
              "'Open Sans'",
              "Arial",
              "Helvetica",
              "sans-serif",
            ],
            "fontSize": "32px",
            "fontWeight": 600,
            "letterSpacing": "0.02em",
            "lineHeight": "40px",
          },
          "h2": Object {
            "color": "#414141",
            "fontFamily": Array [
              "'Open Sans'",
              "Arial",
              "Helvetica",
              "sans-serif",
            ],
            "fontSize": "32px",
            "fontWeight": 400,
            "letterSpacing": "0.02em",
            "lineHeight": "40px",
          },
          "h3": Object {
            "color": "#414141",
            "fontFamily": Array [
              "'Open Sans'",
              "Arial",
              "Helvetica",
              "sans-serif",
            ],
            "fontSize": "22px",
            "fontWeight": 600,
            "letterSpacing": "0.02em",
            "lineHeight": "30px",
          },
          "h4": Object {
            "color": "#414141",
            "fontFamily": Array [
              "'Open Sans'",
              "Arial",
              "Helvetica",
              "sans-serif",
            ],
            "fontSize": "22px",
            "fontWeight": 400,
            "letterSpacing": "0.02em",
            "lineHeight": "30px",
          },
          "h5": Object {
            "fontFamily": Array [
              "'Open Sans'",
              "Arial",
              "Helvetica",
              "sans-serif",
            ],
            "fontSize": "1.5rem",
            "fontWeight": 400,
            "lineHeight": 1.334,
          },
          "h6": Object {
            "fontFamily": Array [
              "'Open Sans'",
              "Arial",
              "Helvetica",
              "sans-serif",
            ],
            "fontSize": "1.25rem",
            "fontWeight": 500,
            "lineHeight": 1.6,
          },
          "htmlFontSize": 16,
          "overline": Object {
            "fontFamily": Array [
              "'Open Sans'",
              "Arial",
              "Helvetica",
              "sans-serif",
            ],
            "fontSize": "0.75rem",
            "fontWeight": 400,
            "lineHeight": 2.66,
            "textTransform": "uppercase",
          },
          "pxToRem": [Function],
          "round": [Function],
          "subtitle1": Object {
            "color": "#414141",
            "fontFamily": Array [
              "'Open Sans'",
              "Arial",
              "Helvetica",
              "sans-serif",
            ],
            "fontSize": "14px",
            "fontWeight": 600,
            "letterSpacing": "0.02em",
            "lineHeight": "20px",
          },
          "subtitle2": Object {
            "color": "#414141",
            "fontFamily": Array [
              "'Open Sans'",
              "Arial",
              "Helvetica",
              "sans-serif",
            ],
            "fontSize": "12px",
            "fontWeight": 600,
            "letterSpacing": "0.02em",
            "lineHeight": "16px",
          },
          "suppressDeprecationWarnings": true,
        },
        "zIndex": Object {
          "appBar": 1100,
          "drawer": 1200,
          "mobileStepper": 1000,
          "modal": 1300,
          "snackbar": 1400,
          "speedDial": 1050,
          "tooltip": 1500,
        },
        Symbol(mui.nested): false,
      }
    }
  >
    <WithStyles(CssBaseline)>
      <CssBaseline
        classes={Object {}}
      />
    </WithStyles(CssBaseline)>
    <WithStyles(HvGrid)
      alignContent="stretch"
      alignItems="stretch"
      component="div"
      container={false}
      direction="row"
      item={false}
      justify="flex-start"
      lg={false}
      md={false}
      sm={false}
      wrap="wrap"
      xl={false}
      xs={false}
      zeroMinWidth={false}
    >
      <HvGrid
        alignContent="stretch"
        alignItems="stretch"
        classes={
          Object {
            "align-content-xs-center": "HvGrid-align-content-xs-center-14",
            "align-content-xs-flex-end": "HvGrid-align-content-xs-flex-end-16",
            "align-content-xs-flex-start": "HvGrid-align-content-xs-flex-start-15",
            "align-content-xs-space-around": "HvGrid-align-content-xs-space-around-18",
            "align-content-xs-space-between": "HvGrid-align-content-xs-space-between-17",
            "align-items-xs-baseline": "HvGrid-align-items-xs-baseline-13",
            "align-items-xs-center": "HvGrid-align-items-xs-center-10",
            "align-items-xs-flex-end": "HvGrid-align-items-xs-flex-end-12",
            "align-items-xs-flex-start": "HvGrid-align-items-xs-flex-start-11",
            "container": "HvGrid-container-2",
            "direction-xs-column": "HvGrid-direction-xs-column-5",
            "direction-xs-column-reverse": "HvGrid-direction-xs-column-reverse-6",
            "direction-xs-row-reverse": "HvGrid-direction-xs-row-reverse-7",
            "grid-lg-1": "HvGrid-grid-lg-1-75",
            "grid-lg-10": "HvGrid-grid-lg-10-84",
            "grid-lg-11": "HvGrid-grid-lg-11-85",
            "grid-lg-12": "HvGrid-grid-lg-12-86",
            "grid-lg-2": "HvGrid-grid-lg-2-76",
            "grid-lg-3": "HvGrid-grid-lg-3-77",
            "grid-lg-4": "HvGrid-grid-lg-4-78",
            "grid-lg-5": "HvGrid-grid-lg-5-79",
            "grid-lg-6": "HvGrid-grid-lg-6-80",
            "grid-lg-7": "HvGrid-grid-lg-7-81",
            "grid-lg-8": "HvGrid-grid-lg-8-82",
            "grid-lg-9": "HvGrid-grid-lg-9-83",
            "grid-lg-auto": "HvGrid-grid-lg-auto-73",
            "grid-lg-true": "HvGrid-grid-lg-true-74",
            "grid-md-1": "HvGrid-grid-md-1-61",
            "grid-md-10": "HvGrid-grid-md-10-70",
            "grid-md-11": "HvGrid-grid-md-11-71",
            "grid-md-12": "HvGrid-grid-md-12-72",
            "grid-md-2": "HvGrid-grid-md-2-62",
            "grid-md-3": "HvGrid-grid-md-3-63",
            "grid-md-4": "HvGrid-grid-md-4-64",
            "grid-md-5": "HvGrid-grid-md-5-65",
            "grid-md-6": "HvGrid-grid-md-6-66",
            "grid-md-7": "HvGrid-grid-md-7-67",
            "grid-md-8": "HvGrid-grid-md-8-68",
            "grid-md-9": "HvGrid-grid-md-9-69",
            "grid-md-auto": "HvGrid-grid-md-auto-59",
            "grid-md-true": "HvGrid-grid-md-true-60",
            "grid-sm-1": "HvGrid-grid-sm-1-47",
            "grid-sm-10": "HvGrid-grid-sm-10-56",
            "grid-sm-11": "HvGrid-grid-sm-11-57",
            "grid-sm-12": "HvGrid-grid-sm-12-58",
            "grid-sm-2": "HvGrid-grid-sm-2-48",
            "grid-sm-3": "HvGrid-grid-sm-3-49",
            "grid-sm-4": "HvGrid-grid-sm-4-50",
            "grid-sm-5": "HvGrid-grid-sm-5-51",
            "grid-sm-6": "HvGrid-grid-sm-6-52",
            "grid-sm-7": "HvGrid-grid-sm-7-53",
            "grid-sm-8": "HvGrid-grid-sm-8-54",
            "grid-sm-9": "HvGrid-grid-sm-9-55",
            "grid-sm-auto": "HvGrid-grid-sm-auto-45",
            "grid-sm-true": "HvGrid-grid-sm-true-46",
            "grid-xl-1": "HvGrid-grid-xl-1-89",
            "grid-xl-10": "HvGrid-grid-xl-10-98",
            "grid-xl-11": "HvGrid-grid-xl-11-99",
            "grid-xl-12": "HvGrid-grid-xl-12-100",
            "grid-xl-2": "HvGrid-grid-xl-2-90",
            "grid-xl-3": "HvGrid-grid-xl-3-91",
            "grid-xl-4": "HvGrid-grid-xl-4-92",
            "grid-xl-5": "HvGrid-grid-xl-5-93",
            "grid-xl-6": "HvGrid-grid-xl-6-94",
            "grid-xl-7": "HvGrid-grid-xl-7-95",
            "grid-xl-8": "HvGrid-grid-xl-8-96",
            "grid-xl-9": "HvGrid-grid-xl-9-97",
            "grid-xl-auto": "HvGrid-grid-xl-auto-87",
            "grid-xl-true": "HvGrid-grid-xl-true-88",
            "grid-xs-1": "HvGrid-grid-xs-1-33",
            "grid-xs-10": "HvGrid-grid-xs-10-42",
            "grid-xs-11": "HvGrid-grid-xs-11-43",
            "grid-xs-12": "HvGrid-grid-xs-12-44",
            "grid-xs-2": "HvGrid-grid-xs-2-34",
            "grid-xs-3": "HvGrid-grid-xs-3-35",
            "grid-xs-4": "HvGrid-grid-xs-4-36",
            "grid-xs-5": "HvGrid-grid-xs-5-37",
            "grid-xs-6": "HvGrid-grid-xs-6-38",
            "grid-xs-7": "HvGrid-grid-xs-7-39",
            "grid-xs-8": "HvGrid-grid-xs-8-40",
            "grid-xs-9": "HvGrid-grid-xs-9-41",
            "grid-xs-auto": "HvGrid-grid-xs-auto-31",
            "grid-xs-true": "HvGrid-grid-xs-true-32",
            "item": "HvGrid-item-3",
            "justify-xs-center": "HvGrid-justify-xs-center-19",
            "justify-xs-flex-end": "HvGrid-justify-xs-flex-end-20",
            "justify-xs-space-around": "HvGrid-justify-xs-space-around-22",
            "justify-xs-space-between": "HvGrid-justify-xs-space-between-21",
            "justify-xs-space-evenly": "HvGrid-justify-xs-space-evenly-23",
            "root": "HvGrid-root-1",
            "spacing-xs-15": "HvGrid-spacing-xs-15-25",
            "spacing-xs-16": "HvGrid-spacing-xs-16-26",
            "spacing-xs-24": "HvGrid-spacing-xs-24-27",
            "spacing-xs-30": "HvGrid-spacing-xs-30-28",
            "spacing-xs-32": "HvGrid-spacing-xs-32-29",
            "spacing-xs-40": "HvGrid-spacing-xs-40-30",
            "spacing-xs-8": "HvGrid-spacing-xs-8-24",
            "wrap-xs-nowrap": "HvGrid-wrap-xs-nowrap-8",
            "wrap-xs-wrap-reverse": "HvGrid-wrap-xs-wrap-reverse-9",
            "zeroMinWidth": "HvGrid-zeroMinWidth-4",
          }
        }
        component="div"
        container={false}
        direction="row"
        item={false}
        justify="flex-start"
        lg={false}
        md={false}
        sm={false}
        wrap="wrap"
        xl={false}
        xs={false}
        zeroMinWidth={false}
      >
        <div
          className="HvGrid-root-1"
        />
      </HvGrid>
    </WithStyles(HvGrid)>
  </ThemeProvider>
</HvProvider>
`;
